session.go

```diff
@@ -0,0 +1,160 @@
+package mockoidc
+
+import (
+	"strings"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+)
+
+// TimeFunc is the default function that the token generators will use for the current time
+var TimeFunc = time.Now
```

Comment:
I have a `NowFunc` in the main `mockoidc.go` file now.

I've setup a `MockOIDC.Now()` function that we can use and consume in all our time logic (it is aware of custom `NowFunc` and calls to `MockOIDC.FastForward()` -- So we should use that throughout our codebase when we want to know `time.Now()`

So for us -- I think all our methods that need to know `Now` get a `now time.Time` argument added to their method signatures. Then we'll use the view of time `MockOIDC` has to manage everything.